package.json

```diff
@@ -16,7 +16,7 @@
     "build": "gulp build",
     "cover": "gulp coverage",
     "deploy": "npm run lint && npm run test && scripts/deploy.sh",
-    "lint": "gulp jshint",
+    "lint": "gulp jshint && gulp tslint",
```

Comment:
We still have the tests. But they will go at some point.
On Nov 8, 2015 22:02, "Kanit Wongsuphasawat" <notifications@github.com>
wrote:

> In package.json
> <https://github.com/vega/vega-lite/pull/708#discussion_r44243418>:
>
> > @@ -16,7 +16,7 @@
> >      "build": "gulp build",
> >      "cover": "gulp coverage",
> >      "deploy": "npm run lint && npm run test && scripts/deploy.sh",
> > -    "lint": "gulp jshint",
> > +    "lint": "gulp jshint && gulp tslint",
>
> are we leaving any js code behind. If not, why still running jshint?
>
> Also a minor point that can be done later, if we can just run tslint
> directly, better run it directly?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vega/vega-lite/pull/708/files#r44243418>.
>